tsl/test/expected/compression_ddl.out

```diff
@@ -644,8 +652,9 @@ AS sub;
                 1
 (1 row)
 
+-- SELECT pg_sleep(200000);
```

Comment:
why do we need the sleep here?

Reply:
we don't 🤦‍♀️ removed.

Reply:
https://github.com/kgyrtkirk/pg-dev-tools/tree/main/psql-debug